### ISSUE TITLE
copy pasting from one code block to another removes line breaks

### ISSUE THREAD

**Issue by ianstormtaylor** [facebook/draft-js#407]
It looks like (tested with the example on the website) if you copy all of the code inside of a code block from one to another, the target code block won't have preserved any line breaks.

![image](https://cloud.githubusercontent.com/assets/311752/15445848/b45fca70-1ebe-11e6-868e-1adbe1323ac4.png)

**Comment by hellendag** [comment]
Weird. Is this only happening with code blocks? And this is a purely internal paste, right?

**Comment by ianstormtaylor** [comment]
Yup! I've only seen it with code blocks, and purely internally. (There's a separate problem with external pasting though, from https://github.com/facebook/draft-js/issues/394 .)

**Comment by ianstormtaylor** [comment]
My guess is this has something to do with https://github.com/facebook/draft-js/issues/452 .

**Comment by tamaspap** [comment]
Any updates on this?
